bot/exts/halloween/spooknamerate.py

```diff
@@ -0,0 +1,342 @@
+from bot.constants import Colours
+from bot.constants import Month
+from bot.utils.decorators import in_month
+
+
+logger = logging.getLogger(__name__)
+
+
+class SpookNameRate(Cog):
+    """
+    A game that asks the user to spookify or halloweenify a name that is given everyday.
+
+    It sends a random name everyday. The user needs to try and spookify it to his best ability and
+    send that word back using the `spooknamerate add entry` command
+
+    Parameters:
+        - bot: A discord.ext.commands.bot.Bot instance which represents the Discord Bot
+
+    Returns:
+        - None
+    """
+
+    def __init__(self, bot: Bot) -> None:
+        self.bot = bot
+        self.messages = {}  # a variable to store all the messages
+        self.emojis_val = {  # the value of the emojis
+            '\N{Jack-O-Lantern}': 1,
+            '\N{Ghost}': 2,
+            '\N{Skull and Crossbones}': 3,
+            '\N{Zombie}': 4,
+            '\N{Face Screaming In Fear}': 5
+        }
+
+        json_data = self.load_json(Path('bot', 'resources', 'halloween', 'spooknamerate.json'))  # load the JSON
+        self.first_names = json_data['first_names']
+        self.last_names = json_data['last_names']
+        # the names are from https://www.mockaroo.com/
+        self.added_messages = ["Let's see if you win?", ":jack_o_lantern: SPOOKY :jack_o_lantern:",
+                               "If you got it, haunt it.", "TIME TO GET YOUR SPOOK ON! :skull:"]
+
+        self.first_time = True
+        self.poll = False
+        self.channel_id = Channels.seasonalbot_commands
+        self.announce_word.start()
+        self.word = None
+
+    @in_month(Month.OCTOBER)
+    @commands.group(name="spooknamerate", invoke_without_command=True)
+    async def spook_name_rate(self, ctx: Context) -> None:
+        """Get help on the Spook Name Rate game."""
+        # Send an embed with help on the command
+
+        help_embed = discord.Embed(
+            title="Spook Name Rate",
+            description=f"Help on the `{self.bot.command_prefix}spooknamerate` command",
+            color=Colours.soft_orange,
+        )
+
+        emoji_message = ""
+        for emoji in self.emojis_val:
+            emoji_message += f"- {emoji} {self.emojis_val.get(emoji)}\n"
+
+        help_embed.add_field(
+            name="How to play",  # display messages on HOW TO PLAY
+            value=f"""Everyday, the bot will post a random word, which you will need to spookify using your creativity.
+You can rate each message according to how scary it is
+At the end of the day, the author of the message with most reactions will be the winner of the day.
+On a scale of 1 to {len(self.emojis_val)}, the reactions order:
+{emoji_message}""",
+            inline=False
+        )
+
+        # command to add spookified name
+        help_embed.add_field(
+            name="How do I add my spookified word?",
+            value=f"Just simply type `{self.bot.command_prefix}spooknamerate add my word`",
```

Comment:
```suggestion
            value=f"Simply type `{self.bot.command_prefix}spooknamerate add my word`",
```
This sounds better in my head at least